crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -52,8 +62,7 @@ pub struct AstScanner<'me> {
   symbols: &'me mut AstSymbols,
   current_stmt_info: StmtInfo,
   result: ScanResult,
-  esm_export_keyword: Option<Span>,
```

Comment:
We need these report some errors in the future. See https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.js%00export+const+a+%3D+1%0Amodule.exports+%3D+%7B%7D&b=%00foo.cjs%00module.exports.foo+%3D+1&o=--bundle+--format%3Desm+--external%3Aext+--outdir%3Ddist